test.js

```diff
@@ -1,9 +1,29 @@
-module.exports = {
-  extends: './eslint.js',
+const eslint = require('./eslint.js');
+const globals = require('globals');
+const mochaPlugin = require('eslint-plugin-mocha');
+
+const testConfig = {
+  languageOptions: {
+    globals: {
+      ...globals.mocha
+    },
+  },
   rules: {
     'no-unused-expressions': 'off'
   },
-  env: {
-    mocha: true
+};
+
+const mochaConfig = {
+  ...mochaPlugin.configs.flat.recommended,
+  rules: {
```

Comment:
this will overwrite the `rules` in `mochaPlugin.configs.flat.recommended`, I guess this is not expected.

Reply:
Yes, this override is correct. I read the [rules in README](https://github.com/lo1tuma/eslint-plugin-mocha?tab=readme-ov-file#rules) and configured the overrides to ignore specific settings in our test code. For example, we use `allow functions` in our test code, but in the eslint-plugin-mocha recommended configuration, this is a warning by [default](https://github.com/lo1tuma/eslint-plugin-mocha/blob/main/docs/rules/no-mocha-arrows.md).